### ISSUE TITLE
Correction, it is "BTW I use arch"

### ISSUE THREAD

**Issue by ejazrasool** [oakes/vim_cubed#39]
I wonder how this cube is rotating having this huge bug.

**Comment by Patsonical** [comment]
Nope, can confirm, it is "I use arch btw".

Source: I use arch (btw)

**Comment by UltiRequiem** [comment]
I can also confirm it.
I use arch btw

**Comment by Purgator** [comment]
According to google photos, it is BTW... I use Arch.
Source : google photos.

What wonder is letting people know on the first place.

I use Arch btw...

**Comment by vereena0x13** [comment]
Yeah idk what ya'll are talking about but uh, I've been daily-driving arch for like a decade plus and, uh, actually, it is

I use arch btw

So. Just for the record. So you know.

**Comment by UltiRequiem** [comment]
> Yeah idk what ya'll are talking about but uh, I've been daily-driving arch for like a decade plus and, uh, actually, it is
> 
> I use arch btw
> 
> So. Just for the record. So you know.

Stop lying. You clearly do not use arch.

**Comment by vereena0x13** [comment]
> > Yeah idk what ya'll are talking about but uh, I've been daily-driving arch for like a decade plus and, uh, actually, it is
> > I use arch btw
> > So. Just for the record. So you know.
> 
> Stop lying. You clearly do not use arch.

Sorry, I misspoke; I run arch btw.

`uname -a`
`Linux vputer 6.11.5-arch1-1 #1 SMP PREEMPT_DYNAMIC Tue, 22 Oct 2024 18:31:38 +0000 x86_64 GNU/Linux`

**Comment by Purgator** [comment]
Is that legit ?
`Linux arichard 5.15.167.4-microsoft-standard-WSL2 #1 SMP Tue Nov 5 00:21:55 UTC 2024 x86_64 GNU/Linux`

**Comment by vereena0x13** [comment]
> Is that legit ? `Linux arichard 5.15.167.4-microsoft-standard-WSL2 #1 SMP Tue Nov 5 00:21:55 UTC 2024 x86_64 GNU/Linux`

Lol yeah, unlike WSL

**Comment by UltiRequiem** [comment]
@vereena0x13 is not an arch linux user guys, i checked his pc

**Comment by vereena0x13** [comment]
> [@vereena0x13](https://github.com/vereena0x13) is not an arch linux user guys, i checked his pc

her*

![Image](https://github.com/user-attachments/assets/a97512cb-52f8-4663-846e-1340cc39d380)

![Image](https://github.com/user-attachments/assets/01a030c6-bab4-4e8a-96fb-900e9c4f2e45)

(yes i just woke up and got out of bed lmao)

**Comment by Purgator** [comment]
@vereena0x13 You are so legit. Mostly because of the middle finder I can only respect you.
I would just advice a bit of accent on the keyboard but nice :D

**Comment by UltiRequiem** [comment]
ok, she uses archlinux guys
